srcpkgs/uucp/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'uucp'
+pkgname=uucp
+version=1.07
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr --mandir=/usr/share/man --with-newconfigdir=/etc/uucp --sbindir=/usr/bin"
+makedepends="wget"
+short_desc="Taylor UUCP is a free implementation of UUCP"
+maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://www.gnu.org/software/uucp/uucp.html"
+distfiles="${GNU_SITE}/$pkgname/$pkgname-$version.tar.gz"
+checksum=060c15bfba6cfd1171ad81f782789032113e199a5aded8f8e0c1c5bd1385b62c
+patch_args="-Np1"
+
+system_accounts="uucp"
+uucp_homedir="/var/spool/uucp"
+
+make_dirs="
+	/var/spool/uucp		0750 uucp uucp
+	/var/spool/uucppublic	0750 uucp uucp"
+
+pre_configure() {
+	sed -i 's|${CC-cc} -E|${CC-cc} -E -O2|g' configure
```

Comment:
Why is -O2 important for preprocesing?

Reply:
It is used for optimization.

Reply:
-E means textual inclusion of headers and expanding macros. There are no optimizations to apply.

Reply:
Removed -02.